src/external/rawspeed/data/cameras.xml

```diff
@@ -5585,6 +5585,17 @@
 		<Crop x="4" y="0" width="-36" height="0"/>
 		<Sensor black="1" white="15868"/>
 	</Camera>
+        <Camera make="RICOH IMAGING COMPANY, LTD." model="PENTAX K-3 II">
+                <ID make="Pentax" model="K-3 II">PENTAX K-3 II</ID>
+                <CFA width="2" height="2">
+                        <Color x="0" y="0">RED</Color>
+                        <Color x="1" y="0">GREEN</Color>
+                        <Color x="0" y="1">GREEN</Color>
+                        <Color x="1" y="1">BLUE</Color>
+                </CFA>
+                <Crop x="4" y="0" width="-36" height="0"/>
+                <Sensor black="1" white="15865"/>
```

Comment:
I think you mentioned different ISOs having different black or white points. That can also be set, just see the Samsung NX1 for example. However judging by the color matrix the sensor is the same as the original K-3 and we didn't do that for that one so maybe it's not really relevant. How big are the differences?